web/content/release/foreman-3.0.adoc

```diff
@@ -0,0 +1,479 @@
+= Foreman 3.0 documentation
```

Comment:
File naming: should it be `web/content/release/foreman/3.0.adoc`?

Reply:
I actually simplified this in the final version.